lib/reply.js

```diff
@@ -197,7 +202,7 @@ function onSendHook (reply, payload) {
 
 function wrapOnSendEnd (err, request, reply, payload) {
   if (err != null) {
```

Comment:
Strict check here?

Reply:
There is no sensible difference in the performances between `err !== undefined && err !== null` and `err != null`. I prefer to use the second, same concept, less typing :D

Reply:
I see, these are nice to know thanks! :)